projects/plugins/boost/app/lib/Status.php

```diff
@@ -38,6 +38,12 @@ public function __construct( $slug ) {
 			Cloud_CSS::get_slug() => array(
 				Critical_CSS::get_slug(),
 			),
+			'minify_css'          => array(
+				'minify',
+			),
+			'minify_js'           => array(
+				'minify',
+			),
 		);
 	}
```

Comment:
Could we use the slug from the class here?
```suggestion
			Cloud_CSS::get_slug()  => array(
				Critical_CSS::get_slug(),
			),
			Minify_CSS::get_slug() => array(
				Minify::get_slug(),
			),
			Minify_JS::get_slug()  => array(
				Minify::get_slug(),
			),
		);
	}
```

Reply:
Good idea!

Reply:
I just noticed that the suggestion has an additional line that caused the PHP error to happen. It's also missing the use statements for the classes.